examples/memory_agentic_chatbot/intelligent-agent.yaml

```diff
@@ -0,0 +1,12 @@
+---
+!wiseagents.agents.CollaboratorWiseAgent
```

Comment:
Note that https://github.com/wise-agents/wise-agents/pull/292 removes the `CollaboratorWiseAgent` since this implementation is no longer needed after the changes made there.

Reply:
Oki, it's great for the "standard" chatbot experience. I'll duplicate it with another name if you haven't anything against it. In the utilty module

Reply:
Ok that's fine.